### DISCUSSION TITLE
Typo in manual

### DISCUSSION THREAD

**Discussion by faragher** [markqvist/Reticulum#337] [General]
I got a 404 on the issue tracker. Don't even know what to do with that. 

Section 4.6.5, Cryptographic Primitives (page 41) version 0.5.5 Beta, publish date 13JUN2023:

Second bullet point reads: X22519 for ECDH key exchanges

I assume this is meant to be X25519 based the source code.

Edit: I just realized I could make the change in the source myself, but I don't actually know how you compiled that. It looks like LaTeX with an intermediary markup layer.

**Comment by markqvist** [comment]
Aha, thanks! Fixed it. Yes, it uses Sphinx to build the various output formats (HTML, PDF, EPUB) via LaTeX. The source files are in https://github.com/markqvist/Reticulum/tree/master/docs/source and there are `make` targets for compiling all the various output formats already (you just need Sphinx installed).
